main.js

```diff
@@ -123,7 +127,7 @@ define([
 				s = new Solver(_equations);
```

Comment:
Should this be solver defined in line 125?

Reply:
I had to put it in try catch block and access it outside it as well. #JavaHabits die hard!